File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    int space = 0;
    int row = 0;
    int hash = 0;
    do
    {
        height = get_int("Height: "); // ask for the heigth
    }
    while (1 > height || height > 8);

    do
    {

        space = height - (row + 1); // prints the spaces
        for (int i = 0; i < space; i++)
        {
            printf(" ");
        }

        hash = row + 1; // prints the first pyramid
        for (int j = 0; j < hash; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int k = 0; k < hash; k++)
        {
            // prints the second pyramid
            printf("#");
        }

        printf("\n");
        row++;

    }
    while (height > row);
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def pyramid(height):
    space = 0
    row = 0
    hash = 0

    while (row < height):
        space = height - (row + 1)
        result = ""                     # String of each line
        for i in range(0, space):       # Add spaces
            result += " "

        hash = row + 1

        for j in range(0, hash):        # Add hashes
            result += "#"

        result += "  "

        for k in range(0, hash):        # Add hashes
            result += "#"

        row += 1
        print(result)                   # Print the line


if __name__ == "__main__":

    height = get_int("Height: ")
    while ((height < 1 or height > 8)):
        height = get_int("Height: ")
    pyramid(height)